templates/web/makemyisland/report/new/notes.html

```diff
@@ -0,0 +1,11 @@
+<p>[% loc("Please note:") %]</p>
+
+<ul class="plain-list">
+    <li>We will only use your personal information in accordance with our <a href="/privacy" target="_blank">privacy policy.</a></li>
+  [% IF c.cobrand.moniker != 'zurich' %]
+    <li>[% loc("Please be polite, concise and to the point.") %]</li>
+    <li>[% loc("Please do not be abusive&nbsp;&mdash; abusing your council devalues the service for all users.") %]</li>
+    <li>[% loc("Writing your message entirely in block capitals makes it hard to read, as does a lack of punctuation.") %]</li>
+  [% END %]
+    <li>[% loc("Remember that FixMyStreet is primarily for reporting physical problems that can be fixed. If your problem is not appropriate for submission via this site remember that you can contact your council directly using their own website.") %]</li>
```

Comment:
FMS ref here - is this line needed? Do we want inline tips like fixmystreet.com has, there's already a link to privacy in the main column.